lib/puppet/provider/user/aix.rb

```diff
@@ -29,6 +29,7 @@
 
   # Provider features
   has_features :manages_aix_lam
```

Comment:
It looks like you can have either `:manages_aix_lam` _or_ `:manages_attributes` -- does that work with the commentary in the commit message around these features?

Reply:
I don't understand what you mean by `either manages_aix_lam or manages_attributes`. The code here indicates that you must now only have the `manages_attributes` feature to use the `attributes` property while before, you had to have `manages_aix_lam`.

Reply:
But the code still says that it has the :manages_aix_lam feature... the comment says you need the :manage_attributes feature.

Reply:
Still don't understand -- the attributes property only requires the `manages_attributes` feature now per https://github.com/puppetlabs/puppet/pull/7115/commits/a065f38e4070ace29981c03d959f488e688d22d2#diff-b6329e40631972a42a3855b62cfb9733R625

We still need `manages_aix_lam` for other params, like the `ia_load_module` param here: https://puppet.com/docs/puppet/5.3/types/user.html#user-attribute-ia_load_module

Reply:
Aha! It wasn't clear to me from the commit comment that both were still in use. Makes sense now.

Reply:
I can reword that part of the message.

Reply:
You got it. Thanks.